build/build.ps1

```diff
@@ -29,13 +24,16 @@ function Build-One {
     }  else {
         $args = @();
     }
-    dotnet build (Join-Path $PSScriptRoot $project) `
-        -c $Env:BUILD_CONFIGURATION `
-        -v $Env:BUILD_VERBOSITY `
-        @args `
-        /property:Version=$Env:ASSEMBLY_VERSION `
-        /property:InformationalVersion=$Env:SEMVER_VERSION `
-        /property:TreatWarningsAsErrors=true
+    $projectFilePath = $(Join-Path $PSScriptRoot $project)
+    $command = "dotnet build $projectFilePath -c $($Env:BUILD_CONFIGURATION) -v $($Env:BUILD_VERBOSITY) $($args) /property:Version=$($Env:ASSEMBLY_VERSION) /property:InformationalVersion=$($Env:SEMVER_VERSION) /property:TreatWarningsAsErrors=true /p:Platform='Any CPU'"
+
+    if ($useVcVars -and $IsWindows) {
+        $command = ". $(Join-Path $PSScriptRoot vcvars.ps1);$command"
```

Comment:
Looks like this script is only used by pyqir so wondering if should really live in that folder.
Also, I don't see it mentioned in the README.md of that folder, so wondering how other developers will find it when trying to build locally.